doc/ADR/homebrew_tap.md

```diff
@@ -14,6 +21,18 @@
    $ brew install diffblue/cbmc/cbmc@5.54.0
    ```
 
+3. To remove the version of CBMC installed you need to issue
```

Comment:
Why are these steps numbered rather than just bullet points? Numbered steps imply you should be following them all, but if I understand this correctly then if I follow all the steps, I'd just end up back where I started.

Reply:
I changed them to be bullet points.

They were originally numbered because they corresponded to the steps needed to install CBMC through the tap.

As I expanded them, they expanded in scope and no longer correspond to a linear sequence of instructions (as they contain instructions for removing, etc)